src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -0,0 +1,35 @@
+package io.iohk.ethereum.db.dataSource
+
+import java.nio.ByteBuffer
+import java.util.concurrent.atomic.AtomicLong
+
+import io.iohk.iodb.{ByteArrayWrapper, LSMStore}
+
+class IodbDataSource(lSMStore: LSMStore) extends DataSource {
+
+  import IodbDataSource._
+
+  override def get(namespace: Byte, key: Key): Option[Value] =
+    lSMStore.get(ByteArrayWrapper((namespace +: key).toArray)).map(v => v.data.toIndexedSeq)
+
+  override def update(namespace: Byte, toRemove: Seq[Key], toUpdate: Seq[(Key, Value)]): DataSource = {
```

Comment:
What will happen if `Key`s in `toUpdate` will have a varying length? Do we need to handle that?